components/HotspotsList.js

```diff
@@ -86,6 +86,11 @@ const HotspotsList = ({ hotspots, rewardsLoading, hotspotsLoading }) => {
     },
   ]
 
+  const [pageSize, setPageSize] = useState(5)
+  const handleTableChange = (pagination, filter, sorter) => {
+    setPageSize(pagination.pageSize)
+  }
```

Comment:
Hi,

Is there a reason why you're changing the default page size from 10 to 5?
Also, `filter` and `sorter` are not used in that function, so it might be cleaner to remove them, although there are no guidelines about that on this repo yet.

```suggestion
  const [pageSize, setPageSize] = useState(10)
  const handleTableChange = ({ pageSize }) => setPageSize(pageSize)
```